compose/index.md

```diff
@@ -29,7 +29,7 @@ anywhere.
 2. Define the services that make up your app in `docker-compose.yml`
 so they can be run together in an isolated environment.
 
-3. Run `docker compose up` and the [Docker compose command](#compose-v2-and-the-new-docker-compose-command) starts and runs your entire app. You can alternatively run `docker-compose up` using the docker-compose binary.
+3. Run `docker compose up` and the [Docker compose command](#compose-v2-and-the-new-docker-compose-command) starts and runs your entire app. You can alternatively run `docker-compose up` using Compose standalone(docker-compose binary).
```

Comment:
Missing also in this file:
- https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/index.md?plain=1#L135
- Not 100% for that one https://github.com/docker/docker.github.io/blob/b76954565d6126734805207c0860c8e46a184897/compose/index.md?plain=1#L140
- https://github.com/docker/docker.github.io/blob/9ecfa4627d7b7a498a4179db94e73d9eec250251/compose/index.md?plain=1#L193

Reply:
I'm also unclear if on line 140 we mean the command or the binary.

Reply:
For the line 140, you're right it seems to mention the binary not the command